### PR TITLE
Milestone 4 - Calculations

### PR DESCRIPTION
**Pull request by RaminMammadzada**
- Add calculate method to organize logic.
- Add operate method to calculate the result
- Import BigJS to calculate big numbers.
- Import calculate method into App component for later use.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @RaminMammadzada  👋

**You've done pretty well here. Nice idea using a short switch statement. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  }
+
+  if (operation === 'x') {
+    return firstValue * secondvalue;
+  }
+
+  if (operation === '/') {
+    if (parseInt(secondvalue, 10) === 0) { return "You can't divide a number by zero"; }
+    return firstValue / secondvalue;
+  }
```
Comment (mikenath223) [inline src/logic/operate.js]: - [x] Please use the Big.js math operators(plus, minus, times, div and modifier), not the regular math operations. Eg: `firstValue.div(secondValue)`

- [x] In addition, please include a case for the percentage operator(%) dividing the number by 100.

Comment (RaminMammadzada) [reply src/logic/operate.js]: Done.

**Review by bellom — APPROVED** [review]
## Hi @RaminMammadzada  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**